scripts/custom/drupal-install-site-3-provision-content.sh

```diff
@@ -18,12 +18,12 @@ DRUSH_ALIAS="${DRUSH_ALIAS:-}"
 # Use local or global Drush, giving priority to a local drush.
 drush="$(if [ -f "${APP}/vendor/bin/drush" ]; then echo "${APP}/vendor/bin/drush"; else command -v drush; fi)"
 
-echo "  > Provision content."
-$drush ${DRUSH_ALIAS} ev -v "require_once '/app/docroot/themes/contrib/civictheme/civictheme.provision.inc'; civictheme_provision_cli();"
-
 echo "  > Provision default content."
 $drush ${DRUSH_ALIAS} -y pm-enable civictheme_content
 
+echo "  > Provision content."
```

Comment:
Needed as default_content module does not handle uuid (duplicate/update)
![Screenshot 2022-07-07 at 10 00 10 AM](https://user-images.githubusercontent.com/83997348/177691497-03d48a92-9ffd-4bf0-9de1-289eab978112.png)

Reply:
@joshua-salsadigital 
but this is an exact problem we need to solve. Please revert this change.

We need to provision as compulsory, but installing the `civictheme_content` module is optional.

Can you please add a hook install to the `civictheme_content` module to clear blocks (by name) before they are crated.